scalameta/semantic/shared/src/main/scala/scala/meta/semantic/Database.scala

```diff
@@ -20,8 +20,8 @@ import scala.meta.{semantic => m}
   lazy val denotations: Map[Symbol, Denotation] = entries.flatMap(_._2.denotations).toMap
   lazy val sugars: Map[Position, String] = entries.flatMap(_._2.sugars).toMap
 
-  def save(classpath: Classpath, sourcepath: Sourcepath): Unit = {
-    this.toSchema(sourcepath).toVfs(classpath).save()
+  def save(classpath: Classpath, sourceroot: AbsolutePath): Unit = {
```

Comment:
`classpath` here should be `AbsolutePath` instead. This fixes another ugliness in the design that I disliked. Thank you!